package.json

```diff
@@ -15,8 +18,7 @@
     "cordova:platform"
   ],
   "scripts": {
-    "cover": "npm run test:coverage",
-    "test": "npm run eslint && npm run test:component && npm run test:objc",
+    "test": "npm run eslint && npm run test:coverage",
```

Comment:
Did we intentionally drop the Objective-C test cases here?

Reply:
Yes, I dropped it intentionally. test:coverage runs this config file. 

https://github.com/apache/cordova-osx/blob/01b31dc9a01321ee5aeba781448cb7359b440ca2/tests/spec/coverage.json

It covers both component and objc tests.

```
"component/**/*[sS]pec.js",
"objc/**/*[sS]pec.js"
```

Reply:
Ahh, okay